library/core/src/num/nonzero.rs

```diff
@@ -287,6 +287,130 @@ nonzero_integers_div! {
     NonZeroUsize(usize);
 }
 
+macro_rules! nonzero_integers_add {
+    ( $( $Ty: ident($Int: ty); )+ ) => {
+        $(
+
```

Comment:
Very very small nit, you put start/end blank spaces on 2 out of 3 macros only, and other macros in the file don't seem to use any ?

Reply:
No worries for the nit :) But I'm sorry I'm not sure which whitespace you are referring to. I've copy-pasted the macro from the `nonzero_integer_div` macro (line 250), and I thought the formatting was unchanged, can you elaborate?

Reply:
I'm speaking about the vertical space between the $( and the impl block, though you can mark this as resolved if it's used elsewhere in the file, I must have missed it

Reply:
Oh, *vertical* space, right. This is something I did not control, thank you :) Fixed!